docs/Job-reference.md

```diff
@@ -5215,15 +5215,20 @@ job {
 job {
     publishers {
         stashNotifier {
+            stashServerBaseUrl(String stashServerBaseUrl) // optional
+            stashUserName(String stashUserName) // optional
+            stashUserPassword(String stashUserPassword) // optional
             commitSha1(String commitSha1) // optional
+            ignoreUnverifiedSSLPeer(boolean ignoreUnverifiedSSLPeer = true) // optional, defaults to false if omitted
             keepRepeatedBuilds(boolean keepRepeatedBuilds = true) // optional, defaults to false if omitted
+            commitSha1(String commitSha1) // optional
         }
     }
 }
 ```
 
 Supports the [Stash Notifier Plugin](https://wiki.jenkins-ci.org/display/JENKINS/StashNotifier+Plugin).
-Uses global Jenkins settings for Stash URL, username, password and unverified SSL certificate handling.
+Uses global Jenkins settings for Stash URL, username, password and unverified SSL certificate handling when not given.
```

Comment:
Add a link to the new "Handling Credentials" page, see https://github.com/jenkinsci/job-dsl-plugin/blob/e917f45419ca2cbcf4cb95e673acdda6667eeeef/docs/Job-reference.md#slack-publisher as an example.